Annotation/declarative_golden_set.json

```diff
@@ -385,7 +409,7 @@
                     }

```

Comment:
I know you've mentioned you don't think we should include stuff like "John writes: xx" however, I think we should because to me this is an embedded clause